include/picongpu/fields/MaxwellSolver/Yee/Yee.hpp

```diff
@@ -152,18 +152,28 @@ namespace picongpu
 
     namespace traits
     {
-        template<class CurlE, class CurlB>
```

Comment:
Note: I think this had no reason to be in the .def file to begin with. It just happened to work before, and wouldn't now.